lib/dogma/rules.ex

```diff
@@ -7,6 +7,7 @@ defmodule Dogma.Rules do
   alias Dogma.Formatter
   alias Dogma.Script
 
+  @default_rule_set Dogma.RuleSet.All
```

Comment:
Thanks! I love them for this kind of thing as you can use the name to say why it exists.